### ISSUE TITLE
projects/core/howto/rdbms.rst should be properly formatted

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#6945]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)| @hawkowl reported|
|-|-|
|Trac ID|trac#6945|
|Type|task|
|Created|2014-02-03 17:39:20Z|
|Branch|https://github.com/twisted/twisted/tree/rdbms-docs-format-6945|

As with [#6943](https://github.com/twisted/twisted/issues/6943), this is updating the rST sources to be correctly formatted.

<details><summary>Searchable metadata</summary>

```
trac-id__6945 6945
type__task task
reporter__hawkowl hawkowl
priority__normal normal
milestone__Lore_to_Sphinx Lore to Sphinx
branch__branches_rdbms_docs_format_6945 branches/rdbms-docs-format-6945
branch_author__hawkowl hawkowl
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation documentation
time__1391449160000000 1391449160000000
changetime__1393512618000000 1393512618000000
version__None None
owner__hawkowl hawkowl

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_1"></a>|@hawkowl commented|
|-|-|

(In [41540]) Branching to rdbms-docs-format-6945.

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_2"></a>|@hawkowl removed owner|
|-|-|

Putting this up for review.

Most of the changes are formatting, although I did fix a few spelling mistakes and switched some API links to have consistent naming.

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_3"></a>|@adiroiban commented|
|-|-|

Looks good. Thanks!

**Comment by twisted-trac** [comment]
|[<img alt="wallrj's avatar" src="https://avatars.githubusercontent.com/u/978965?s=50" width="50" height="50">](https://github.com/wallrj)<a name="note_4"></a>|@wallrj set owner to @hawkowl|
|-|-|

Thanks hawkowl.

Looks good. I only spotted one problem.

I ran "make linkcheck" and got this -- projects/core/howto/rdbms.rst:21: [broken] http://www.amk.ca/python/writing/DB-API.html: HTTP Error 404: Not Found

Looks like adiroban also read through it so please merge after fixing or
removing that link.

-RichardW.

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_5"></a>|@hawkowl set status to `closed`|
|-|-|

(In [41769]) Merge rdbms-docs-format-6945

Author: hawkowl
Reviewers: adiroiban, rwall
Fixes: [#6945](https://github.com/twisted/twisted/issues/6945)

projects/core/howto/rdbms.rst is now a properly formatted rST document.
